tools/static-assets/skel-full/imports/api/links/server/publications.tests.js

```diff
@@ -0,0 +1,18 @@
+// Tests for the links publications
```

Comment:
(Like maybe we should include the publication collection package and the validated method stuff for method tests?)